lib/irslackd.js

```diff
@@ -545,6 +551,18 @@ class Irslackd {
     let message = this.meText(verb + ' @ ' + ircReactee + ' :' + event.reaction + ':');
     this.ircd.write(ircUser.socket, ircReacter, 'PRIVMSG', [ ircChan, message ]);
   }
+  async onSlackTyping(ircUser, event) {
+    let [ircNick, ircTarget] = await this.resolveSlackTarget(ircUser, event);
+    if (!ircNick || !ircTarget) {
+      this.logError(ircUser, 'Failed this.resolveSlackTarget; event: ' + util.inspect(event));
+      return;
+    }
+
+    console.log('slack_typing', ircNick, ircTarget);
+
+    let line = this.typingText('1');
+    this.ircd.write(ircUser.socket, ircNick, 'PRIVMSG', [ ircTarget, line ]);
+  }
```

Comment:
Can we get a unit test for this event and the new behavior in `onIrcPrivmsg`?

Reply:
Once the new code is reviewed, I can look into unit testing.